### ISSUE TITLE
Collective processed refunds without Host approval

### ISSUE THREAD

**Issue by alanna** [opencollective/opencollective#6875]
[According to the docs](https://docs.opencollective.com/help/fiscal-hosts/payouts/refunds) only Host admins can process refunds. This is super important to charity hosts especially, because they are not legally allowed to refund certain transactions once a tax deductible receipt has been issued. Further, the host has to be involved to process the payment if it was a bank transfer and other logistical considerations.

The below refunds in [this Collective](https://opencollective.com/heart-place-charity-hospital/transactions) were triggered by the Collective admin alone, without the Host's approval.

<img width="1177" alt="Screen Shot 2023-07-20 at 9 41 51 AM" src="https://github.com/opencollective/opencollective/assets/5498878/4a952570-5bb7-481b-a3f1-860eec0affa3">

We can't see who pressed the refund button easily (it's not on the contribution's page nor the activity log) but Chloe contacted OC support and they confirmed it was the Collective admin who did it and it went through.

**Comment by alanna** [comment]
Apparently the docs were wrong. @Betree says Collective admins can process refunds themselves if:

* The transaction is less than 30 days old
* It's a direct contribution (no added funds)

However, the $980 one above was a manual bank transfer added funds one and it should not have worked.

**Comment by Betree** [comment]
> However, the $980 one above was a manual bank transfer added funds one and it should not have worked.

No, pending contributions that are later confirmed are still considered contributions. Added funds are only the ones added via the "Add funds" form, and can be recognized through the "Added funds" label on the transactions

![image](https://github.com/opencollective/opencollective/assets/1556356/e25946df-46d1-4152-8036-e6e436b0fbc0)

**Comment by Betree** [comment]
To clarify, this behavior is not a bug but something that was purposely done like that to let collective admins reject contributions and fight against spammy contributors (casinos, adult websites, etc). See https://github.com/opencollective/opencollective/issues/3463.

Updating the docs should be our priority. There's also [this section](https://docs.opencollective.com/help/financial-contributors/payments#refunds) to look at.

After that, we can eventually seek to add a host setting to disable collective admins' refunds, but as with any feature request, the use case should be properly explained - ideally in a separate issue.

**Comment by alanna** [comment]
> No, pending contributions that are later confirmed are still considered contributions.

How is the workflow supposed to go when it's a bank transfer? The Collective admin clicks "refund" but then... the money just sits there. The Host admin has no way of knowing this even happened, and the system has no way of transferring money via manual bank transfer. The purpose of not allowing Collectives to refund for added funds is that the Host must be involved to move the money, and this is true whether it was done via added funds or confirming a pending bank transfer.

Due to laws in NZ, Gift Collective needs to have the following policy about refunds (we are legally not allowed to refund charity donations except in certain circumstances):

> Gift Collective is not obliged to make refunds. All refunds will be at the discretion of Gift Collective. In assessing whether refunds of online donations can be facilitated, we will determine whether the donation in question has already been spent by the Collective. If a donation is refunded, the associated tax receipt is no longer valid - it is the donor’s responsibility to only submit correct receipts to Inland Revenue. Gift Collective will refund if: donations made are fraudulent due to unauthorised use of a credit card; an error was made with a donation by our staff or platform; if the donor made an error when submitting; if there are exceptional circumstances. Requests for refunds must be made within 30 days of the donation.

The 30 days thing is already in place, which is great. But how can we get OC to help us with the other aspects? Can Collective admin refunds be turned off for this one Host? Or is there a way for Host admins to have an approval step in the process after Collective admins click refund?

**Comment by Betree** [comment]
It is a good idea to turn off refunds for manual bank transfers until we have a notification process for the host.

> Can Collective admin refunds be turned off for this one Host?

We can add a `host.data.policies.COLLECTIVE_ADMINS_CAN_REFUND` policy.

> Or is there a way for Host admins to have an approval step in the process after Collective admins click refund?

That would be ideal but way more complex to design and implement. I'll keep it out of this issue to resolve it quickly, but we can suggest the idea in a new one.

**Comment by alanna** [comment]
OK great I will follow that other thread for the short term fix, and will leave this open to address the question about turning off refunds for manual bank transfers, and the larger issues.

**Comment by Betree** [comment]
> the question about turning off refunds for manual bank transfers

That's already done as part of https://github.com/opencollective/opencollective/issues/6886.
